sos/report/plugins/kdump.py

```diff
@@ -67,6 +69,11 @@ def setup(self):
                         + "kdump.img"
         if os.path.exists(initramfs_img):
             self.add_cmd_output("lsinitrd %s" % initramfs_img)
+            with tempfile.NamedTemporaryFile(delete=False) as tempfp:
+                p = subprocess.Popen(['lsinitrd', '-f', 'squash/root.img', initramfs_img], stdout=tempfp)
```

Comment:
You can't leave temporary files lying around after an sos run. This either has to be done using a pipe between the `lsinitrd -f` and `unsquashfs -ll` commands, or by using temporary paths that are contained in the `sos` working directory (and so are cleaned up during exit handling).